### DISCUSSION TITLE
Search by tag

### DISCUSSION THREAD

**Discussion by jgarte** [jrnl-org/jrnl#1410] [Q&A] [answered]
Hi,

how can I search by tag?

I know how to exclude tags `not`.

**Comment by wren** [comment, marked as answer]
Hi!

You can search by tag by just typing in the tags you're searching for. Example:

```sh
jrnl @tag1 @tag2 @tag3
```

You can also mix this in with the other search terms, if you'd like:

```sh
jrnl -today-in-history @tag2 -starred -contains "some text"
```
